draw/src/main/java/org/odk/collect/draw/QuitDrawingDialog.kt

```diff
@@ -0,0 +1,50 @@
+package org.odk.collect.draw
+
+import android.app.Activity
+import androidx.appcompat.app.AlertDialog
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import org.odk.collect.draw.databinding.QuitDrawingDialogLayoutBinding
+
+object QuitDrawingDialog {
```

Comment:
I like this having a separate implementation from `QuitFormDialog` as it feels more like they **happen** to be the same as opposed to them always being the same.